casepro/backend/__init__.py

```diff
@@ -199,6 +199,15 @@ def fetch_contact_messages(self, org, contact, created_after, created_before):
                  Message.as_json() for incoming messages and Outgoing.as_json() for outgoing messages.
         """
 
+    @abstractmethod
+    def get_url_patterns(self):
+        """
+        Returns the list of URL patterns that should be registered for this
+        backend.
```

Comment:
Another code convention that you guys either share or have figured out is that we tend to use '' for code strings and "" for display strings - when possible.